streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -85,14 +90,7 @@ public boolean initializeStateStores() {
     }
 
     @Override
-    public void initializeTopology() {
-        //no-op
-    }
-
-    @Override
-    public void initializeTaskTime() {
-        //no-op
-    }
+    public void initializeTopology() {}
```

Comment:
Not really, because we call `initializeTopology()` within `AssignedTasks#transitToRunning(T task)` that does not know the type (it generic `T extends Tasks`) -- hence, we would need to do an `instanceof` call within that method what is undesired.